test/plugins/test_jedi_rename.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Palantir Technologies, Inc.
+import os
+import sys
+
+from test.test_utils import MockWorkspace
+
+import pytest
+from pyls import uris
+from pyls.plugins.jedi_rename import pyls_rename
+from pyls.workspace import Document
+
+LT_PY36 = sys.version_info.major < 3 or (sys.version_info.major == 3 and sys.version_info.minor < 6)
+
+DOC_NAME = 'test1.py'
+DOC = '''class Test1():
+    pass
+
+class Test2(Test1):
+    pass
+'''
+
+
+@pytest.fixture
+def tmp_workspace(workspace):
```

Comment:
Did you copy this fixture from another file? If that's the case, please move it to `test/fixtures.py` and remove it from all files that declare it.

Reply:
yeah it was copy-pasta. will improve the test file

Reply:
We can't move it because it depends on variables declared on this file. Should we create a helper function in ``text/fixtures.py`` (something like `make_tmp_workspace(list_of_files)`), then update all `tmp_workspace` declarations to invoke that function?

Reply:
That would really nice, thanks!